app/dashboard/page.tsx

```diff
@@ -1,11 +1,104 @@
+    })
+  }
+
+  useEffect(() => {
+    getProfileByUserId().then((res) => {
+      console.log(res);
+      setName(res?.full_name || "");
+      setEmail(res?.email || "");
+      setBio(res?.bio || "");
+      setOrg(res?.organizations?.name || "You are not part of an organization. Go to organization overview to create or join an organization.");
+      setLoading(false);
+    }).catch((error) => (displayErrorToast(error)));
+  }, [])
+
+  return (
+    <div className="flex flex-col 2xl:gap-y-12 gap-y-10">
+      <PageHeading header="Profile" description="This is how others will see you on the site" />
+      {loading ?
+        <div className="text-center 2xl:mt-72 mt-52 text-darkmaroon">
+          <div className="text-xl font-medium">Getting Profile data</div>
+          <div className="loadingAnimation"><div></div><div></div><div></div></div>
+        </div>
+        :
+        <div className="flex flex-col 2xl:gap-y-12 2xl:w-[1200px] gap-y-10 w-[645px]">
+          {/* Name */}
+          <div className="h-24 text-wrap">
+            <div className="2xl:text-3xl text-2xl font-medium mb-4">
+              Name
+            </div>
+            <input type="text" minLength={4} value={name} onChange={(e) => { setName(e.target.value) }} className="outline-none w-full bg-inherit 2xl:text-lg text-body border-b border-body" />
+            {name.length == 0 && <p className="text-red-800 mt-1 2xl:text-base text-sm">Name cannot be empty</p>}
+          </div>
+          {/* Email */}
+          <div className="h-24 text-wrap">
+            <div className="2xl:text-3xl text-2xl font-medium mb-4">
+              Display Email
+            </div>
+            <div className="outline-none w-full bg-inherit 2xl:text-lg text-body border-b border-body" >
+              {email}
+            </div>
+          </div>
+          {/* Bio */}
+          <div className="h-44 text-wrap">
+            <div className="2xl:text-3xl text-2xl font-medium mb-4">
+              Bio
+            </div>
+            <textarea required={true} value={bio} rows={4} onChange={(e) => { setBio(e.target.value) }} className="outline-none w-full bg-inherit 2xl:text-lg text-body resize-none border-b border-body" />
+            {bio.length == 0 && <p className="text-red-800 mt-1 2xl:text-base text-sm">Bio cannot empty</p>}
+          </div>
+          {/* Organization */}
+          <div className=" text-wrap">
+            <div className="2xl:text-3xl text-2xl font-medium mb-4">
+              Organization
+            </div>
+            <div className="2xl:text-lg ">
+              {org}
+            </div>
+          </div>
+          {/* Update Profile */}
+          <button disabled={name.length == 0 || !email.match(isValidEmail) || bio.length == 0} onClick={() => handleUpdateProfile()} className="disabled:cursor-not-allowed bg-[#CAA6A6] rounded-lg w-[400px] h-16 hover:bg-[#944E63] text-white text-2xl ">{!updating ? "Update Profile" : "Updating Profile..."}</button>
```

Comment:
<img width="736" alt="Screenshot 2024-02-27 at 10 22 16 PM" src="https://github.com/nam-t24/Senior-CS-Project/assets/71054634/a37b60a6-8581-4552-af9a-3cc696652343">
Could you make the button smaller for smaller screens big jumbo button lol

Reply:
Also could you use tailwind theme extensions i added to tailwind.config for colors since its hard to determine color based on hex code. Like bg-maroon or bg-lightmaroon